### DISCUSSION TITLE
Is it possible to add a block with an image *above* the navbar?

### DISCUSSION THREAD

**Discussion by giabaio** [quarto-dev/quarto-cli#2936] [Q&A]
Basically, the title *is* my question... I would like to create a template/website in which the first element is a block with a background image, which sits on top of the standard navbar.

I think I can tweak the `css` for `.navbar` to include a background image and style it more or less to like like I think it should, but that's a bit (well... a lot, actually!) fiddly and doesn't scale up/down with different resolutions of the screen...

I am not sure whether this would imply some redefinition of the main page appearance or whether it's even possible, but I thought I'd ask...

Thanks

**Comment by dmi3kno** [comment]
Also curious to know. I suspect this can be done with partials, but I am a little unsure how go about it.

**Comment by mcanouil** [comment]
There is no built-in/easy way of doing this currently.

**Reply by cderv** [reply]
Indeed this is not possible. I will move this to a feature request maybe ? 🤔 

This would require a rather big evolution to offer customization for `.ejs` template for any part of a Website creation.
